### DISCUSSION TITLE
Why not use official immer middleware?

### DISCUSSION THREAD

**Discussion by ShinyLeee** [udecode/zustand-x#40] [Q&A] [answered]
Just a little question, [official-immer-middleware](https://github.com/pmndrs/zustand/blob/main/src/middleware/immer.ts#L68) seems more robust? `updater` might be a plain object.

**Comment by zbeyens** [comment, marked as answer]
It was a quick implementation, I'm open to support all cases

**Reply by ShinyLeee** [reply]
Got it, maybe i can do it if I have time recently.
